projects/cdk/directives/control/test/control.directive.spec.ts

```diff
@@ -0,0 +1,47 @@
+import {ChangeDetectionStrategy, Component} from '@angular/core';
+import {ComponentFixture, TestBed} from '@angular/core/testing';
+import {FormControl, FormGroup, ReactiveFormsModule} from '@angular/forms';
+import {By} from '@angular/platform-browser';
+import {TuiControlModule} from '@taiga-ui/cdk';
+import {configureTestSuite} from '@taiga-ui/testing';
+
+describe(`TuiControlDirective`, () => {
+    @Component({
+        template: `
+            <form [formGroup]="form">
```

Comment:
What do you think is it necessary to add an example here?

<img width="219" alt="image" src="https://user-images.githubusercontent.com/12021443/230302153-b8dbbf93-b0c6-4fa8-884a-0a73b0cbe4e4.png">

Reply:
I don't think it's worth it, but I'll add one for tuiValueChanges